src/main/java/uk/nhs/prm/repo/ehrtransferservice/services/ConversationActivityService.java

```diff
@@ -0,0 +1,36 @@
+package uk.nhs.prm.repo.ehrtransferservice.services;
+
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Service
+public final class ConversationActivityService {
+    private final Map<UUID, Instant> conversations;
+
+    public ConversationActivityService() {
+        this.conversations = new ConcurrentHashMap<>();
+    }
+
+    public void captureConversationActivityTimestamp(UUID inboundConversationId) {
```

Comment:
Still learning towards just naming this as `captureConversationActivity` and reflect throughout, not fussed though 😆!